test/test_mkldnn.py

```diff
@@ -388,6 +388,19 @@ def test_zero_(self):
             x1.zero_(),
             x2.zero_().to_dense(),
         )
+        # mkldnn has block format, make sure mkldnn zero_ also work for a
+        # mkldnn tensor which has block format
+        conv2d = torch.nn.Conv2d(3, 7, 2, 3).float()
+        mkldnn_conv2d = mkldnn_utils.to_mkldnn(copy.deepcopy(conv2d))
```

Comment:
Could you add a comment here says mkldnn conv2d's weight is in block format?

Reply:
this step only get plain format(just call cpu_tensor.to_mkldnn()).

Reply:
just run this step: https://github.com/pytorch/pytorch/blob/a6fb6e1fb39f1f4a509dec04e4719ff9e2d04c83/torch/utils/mkldnn.py#L47